passes/techmap/dfflibmap.cc

```diff
@@ -115,7 +122,15 @@ static bool parse_pin(LibertyAst *cell, LibertyAst *attr, std::string &pin_name,
 	return false;
 }
 
-static void find_cell(LibertyAst *ast, IdString cell_type, bool clkpol, bool has_reset, bool rstpol, bool rstval)
+static int glob_match(const char *pattern, const char *string) {
```

Comment:
Please use `patmatch` from `kernel/yosys.cc` for this, if possible

Reply:
I didn't see that, thanks. Opened #4223.